shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx

```diff
@@ -10,7 +10,7 @@ const copyChildren = (children: React.ReactNode): React.ReactNode =>
   // @ts-ignore
   React.Children.map(children, child => (child ? React.cloneElement(child) : child))
 
-export const animationDuration = 1500
+export const animationDuration = 3000
```

Comment:
FWIW I liked the original duration, this feels long to me

Reply:
👍 moved it to 2000.